src/Resources/assets_src/src/js/composer.js

```diff
@@ -1,14 +1,6 @@
 /**
- *
- * This file is part of the Sonata package.
```

Comment:
is it correct, that all headers get removed?

Reply:
@OskarStark i tried to leave only one header comment for files in `Resources/public` directory after running `gulp`. So, i removed all comments about `This file is part of the Sonata` in `Resources/assets_src/src` and modified `Resources/assets_src/gulp/util/banner.js` file.